draft-ietf-httpbis-unprompted-auth.md

```diff
@@ -205,10 +231,26 @@ of bytes necessary.
 
 ## Key Exporter Output {#output}
 
-The output of the exporter is a 32-byte symmetric key. That symmetric key is
-then used as nonce which can be signed using the client's chosen asymmetric
-private key. The resulting signature is then transmitted to the server using
-the Authorization field.
+The TLS key exporter output is described in {{fig-output}}:
+
+~~~
+  Nonce (256),
+  Verification (128),
+~~~
+{: #fig-output title="Key Exporter Output Format"}
+
+The key exporter context contains the following fields:
+
+Nonce:
```

Comment:
I don't like this word, as we've previously discussed.  "Signature Input" would work.

Reply:
Signature Input sounds good to me. I'll address this as an editorial change after I land the current train of PRs to avoid having to deal with 17 merge conflicts

Reply:
Filed #2578 to track this